addons/mail/models/mail_mail.py

```diff
@@ -536,7 +548,7 @@ def _split_by_mail_configuration(self):
             for batch_ids in tools.split_every(batch_size, record_ids):
                 yield mail_server_id, alias_domain_id, smtp_from, batch_ids
 
-    def send(self, auto_commit=False, raise_exception=False):
+    def send(self, auto_commit=False, raise_exception=False, log_callback=None):
```

Comment:
`log_callback` does not mean anything, except that it might be a callback function.  You really need to learn naming things.  Simply ask yourself: what is that function's purpose?  Its purpose is to keep track of mail ids that have been sent.  So we could name it `log_sent`, and this instantly makes the code clearer.